edx_rest_framework_extensions/tests/test_authentication.py

```diff
@@ -167,3 +186,79 @@ def test_authenticate_user_creation_with_request_exception(self):
             self.assertRaises(AuthenticationFailed, self.auth.authenticate, request)
 
         self.assertEqual(User.objects.all().count(), original_user_count)
+
+
+@override_settings(
+    ROOT_URLCONF='edx_rest_framework_extensions.tests.test_authentication',
```

Comment:
This testing is at too high a level. This is more integration testing than unit. You should only need to test that certain exceptions are raised.